docs/src/guide/installation.md

```diff
@@ -1,15 +1,23 @@
 # Installation
 
-Similar to [installing any other registered package](https://julialang.github.io/Pkg.jl/v1/getting-started/)
-in Julia just run from a Julia REPL:
-```
-] add DFTK
+In case you have not yet installed Julia, first do so by
+[downloading the Julia binaries](https://julialang.org/downloads/)
+and following the [Julia installation instructions](https://julialang.org/downloads/platform/).
+At least **Julia 1.4** is required for DFTK.
+
+Afterwards you can install DFTK
+[like any other package](https://julialang.github.io/Pkg.jl/v1/getting-started/)
+in Julia. For example run in your Julia REPL terminal:
+```julia
+using Pkg
+pkg"add DFTK"
 ```
-or if you like to be fully up to date:
+which will install the latest DFTK release.
+Alternatively (if you like to be fully up to date) install the master branch:
 ```
-] add DFTK#master
+using Pkg
+pkg"add DFTK#master"
```

Comment:
We can add something like "if you want to start working on DFTK, you can do `]dev DFTK` which will check out a development copy in (on linux) ~/.julia/dev/DFTK"

Reply:
Good idea. I'll add a few "developer's notes".